packages/babel-core/test/config-chain.js

```diff
@@ -84,16 +84,6 @@ describe("buildConfigChain", function () {
     const expected = [
       {
         options: {
```

Comment:
This diff isn't great, but what moved here was the `.babelignore` ordering. But since we just append `ignore` values together, this change in ordering should essentially be a noop.